main.go

```diff
@@ -1064,7 +1064,7 @@ func parseAccountQuery(queryResponseBytes []byte) (int, int, error) {
 			return 0, 0, fmt.Errorf("sequence number is not an integer")
 		}
 		return accInt, seqInt, nil
-	} else if acctType.Type == "/ethermint.types.v1.EthAccount" {
+	} else if strings.Contains(acctType.Type, "EthAccount") {
```

Comment:
If you want to be stricter, you could go with strings.HasSuffix(). Do you have a few examples of what kind of Eth account types are there?